docs/supported-technologies.asciidoc

```diff
@@ -67,3 +67,13 @@ requests using these libraries:
 - `net/http`
 - Http.rb (v0.6+)
 - Faraday (v0.2.1+)
+
+[float]
+[[supported-technologies-http]]
+=== External HTTP requests
```

Comment:
Did you mean to put these under HTTP Requests? As opposed to something _background jobs_?

Reply:
Indeed, this is wrong. I corrected it to "backgroud processing"